libkpod/config.go

```diff
@@ -0,0 +1,267 @@
+package libkpod
```

Comment:
@14rcole I would go back to using the old libkpod.New() for now. Making the default config more generic (possibly making a specific shared config that applies exclusively to libkpod, and another that contains things CRI-O specific) is a good idea, but we can worry about it in followon PRs - this one is already kind of large